web-ide/proxy/src/docker.ts

```diff
@@ -86,7 +87,7 @@ export default class Docker {
                     cResolve(container)
                 })
             })
-            runner.on('container', (container :Container) => debug(`created container ${container.id}`))
+            runner.on('container', (container :Container) => { debug(`created container ${container.id}`) })
```

Comment:
this debug thing is a small library which only logs when there is a special env variable or command line argument present, otherwise it is ignored.

It's imported at top of the file: `debug = require('debug')('webide:session'),`

I don't need to use it but found it helpful. It is smaller than a full logging library, which is probably overkill...the way we're running this service in the cloud, all we need is standard error/out

Reply:
or in other words 🤷‍♂ javascript